demisto_sdk/commands/init/templates/HelloIAMWorld/HelloIAMWorld_test.py

```diff
@@ -38,267 +38,266 @@ def get_outputs_from_user_profile(user_profile):
+        assert outputs.get('success') is False
+        assert outputs.get('errorCode') == 500
+        assert outputs.get('errorMessage') == 'message: details'
+
+
+class TestCreateUserCommand:
+    def test_create_user_command__success(self, mocker):
+        """
+        Given:
+            - An app client object
+            - A user-profile argument that contains an email of a non-existing user in the application
+        When:
+            - Calling function create_user_command
+        Then:
+            - Ensure a User Profile object with the user data is returned
+        """
+        client = mock_client()
+        args = {'user-profile': {'email': 'testdemisto2@paloaltonetworks.com'}}
+
+        mocker.patch.object(client, 'get_user', return_value=None)
+        mocker.patch.object(IAMUserProfile, 'map_object', return_value={})
+        mocker.patch.object(client, 'create_user', return_value=USER_APP_DATA)
+
+        user_profile = IAMCommand().create_user(client, args)
+        outputs = get_outputs_from_user_profile(user_profile)
+
+        assert outputs.get('action') == IAMActions.CREATE_USER
+        assert outputs.get('success') is True
+        assert outputs.get('active') is True
+        assert outputs.get('id') == 'mock_id'
+        assert outputs.get('username') == 'mock_user_name'
+        assert outputs.get('details', {}).get('first_name') == 'mock_first_name'
+        assert outputs.get('details', {}).get('last_name') == 'mock_last_name'
+
+    def test_create_user_command__user_already_exists(self, mocker):
+        """
+        Given:
+            - An app client object
+            - A user-profile argument that contains an email of a user
+        When:
+            - The user already exists in the application and disabled
+            - allow-enable argument is false
+            - Calling function create_user_command
+        Then:
+            - Ensure the command is considered successful and the user is still disabled
+        """
+        client = mock_client()
+        args = {'user-profile': {'email': 'testdemisto2@paloaltonetworks.com'}, 'allow-enable': 'false'}
+
+        mocker.patch.object(client, 'get_user', return_value=DISABLED_USER_APP_DATA)
+        mocker.patch.object(client, 'update_user', return_value=DISABLED_USER_APP_DATA)
+
+        user_profile = IAMCommand().create_user(client, args)
+        outputs = get_outputs_from_user_profile(user_profile)
+
+        assert outputs.get('action') == IAMActions.UPDATE_USER
+        assert outputs.get('success') is True
+        assert outputs.get('active') is False
+        assert outputs.get('id') == 'mock_id'
+        assert outputs.get('username') == 'mock_user_name'
+        assert outputs.get('details', {}).get('first_name') == 'mock_first_name'
+        assert outputs.get('details', {}).get('last_name') == 'mock_last_name'
+
+
+class TestUpdateUserCommand:
+    def test_update_user_command__non_existing_user(self, mocker):
```

Comment:
update all test function names accordingly
```suggestion
    def test_non_existing_user(self, mocker):
```